# Fix reference to etc_netconfig being in current directory.  Explicitly reference it.
diff --git a/doc/Makefile.am b/doc/Makefile.am
index cc4fa76..bac7c0f 100644
--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -1,5 +1,5 @@
 
 install: install-am
 	$(mkinstalldirs) $(DESTDIR)/etc
-	cp -p ./etc_netconfig $(DESTDIR)/etc/netconfig
+	cp -p $(srcdir)/etc_netconfig $(DESTDIR)/etc/netconfig
 	chmod 0644 $(DESTDIR)/etc/netconfig
